src/main/scala/ch/openolitor/stammdaten/models/AbosModels.scala

```diff
@@ -35,7 +35,7 @@ case class AboId(id: Long) extends BaseId
 object IAbo {
   def calculateAktiv(start: LocalDate, ende: Option[LocalDate]): Boolean = {
     val yesterday = LocalDate.now.minusDays(1)
-    (start.isBefore(LocalDate.now) || start.isEqual(LocalDate.now)) && (ende map (_.isAfter(yesterday)) getOrElse true)
+    !start.isAfter(LocalDate.now) && (ende map (_.isAfter(yesterday)) getOrElse true)
```

Comment:
This still returns different results if we execute in the morning or afternoon if we try with a date XX.YY.2018 12:00.
BTW: This case can very easily be covered with a unit test.

Reply:
You could compare with daysBetween (http://www.joda.org/joda-time/apidocs/org/joda/time/Days.html#daysBetween-org.joda.time.ReadableInstant-org.joda.time.ReadableInstant-).